src/Language/PureScript/Pretty/Values.hs

```diff
@@ -124,7 +124,7 @@ prettyPrintDoNotationElement (DoNotationBind binder val) =
     ]
 prettyPrintDoNotationElement (DoNotationLet ds) =
   concat <$> sequence
-    [ return "let "
+    [ return "let\n"
```

Comment:
So all `let`s are going to be followed by a newline? I think this style should be conditional on the length of `ds`.